internal/services/containers/kubernetes_cluster_resource.go

```diff
@@ -1658,6 +1706,11 @@ func resourceKubernetesClusterRead(d *pluginsdk.ResourceData, meta interface{})
 			return fmt.Errorf("setting `windows_profile`: %+v", err)
 		}
 
+		httpProxyConfig := flattenKubernetesClusterHttpProxyConfig(props, flattenedDefaultNodePool)
+		if err := d.Set("http_proxy_config", httpProxyConfig); err != nil {
+			return fmt.Errorf("setting `http_proxy_config`: %+v", err)
+		}
+
```

Comment:
Would be great if we could move this up to ~line 1675, as the others are alphabetically ordered. It's not done everywhere, but here it is 😄